crates/astria-composer/local.env.example

```diff
@@ -47,6 +47,13 @@ ASTRIA_COMPOSER_MAX_SUBMIT_INTERVAL_MS=2000
 # key and nonce bytes
 ASTRIA_COMPOSER_MAX_BYTES_PER_BUNDLE=200000
 
+# Max amount of `SizedBundle`s to allow to accrue in the `BundleFactory`'s finished queue.
+# ASTRIA_COMPOSER_FINISHED_QUEUE_CAPACITY * ASTRIA_COMPOSER_MAX_BYTES_PER_BUNDLE (e.g.
+# 10 * 200KB) is the limit on how much memory the finished bundle queue can consume.
+# This should be lower than the resource limit enforced by Kubernetes on the pod, defined here:
+# https://github.com/astriaorg/astria/blob/622d4cb8695e4fbcd86456bd16149420b8acda79/charts/evm-rollup/values.yaml#L276
+ASTRIA_COMPOSER_FINISHED_QUEUE_CAPACITY=10
```

Comment:
you should add this in the charts directory for CI. https://github.com/astriaorg/astria/blob/0124b5723d980dabb9e9928eca3c6436f785122b/charts/evm-rollup/templates/configmap.yaml#L83

Reply:
10 might be a bit too small? i think we can calibrate the numbers during testing.